### PR TITLE
pkp/pkp-lib#1144 and pkp/pkp-lib#1258 Editor decision forms

### PR DESCRIPTION
**Pull request by NateWr**

### CODE REVIEW

File: classes/workflow/EditorDecisionActionsManager.inc.php [hunk 1]
```diff
@@ -130,24 +130,18 @@ static function _submissionStageDecisions() {
 				'operation' => 'externalReview',
 				'name' => 'externalReview',
 				'title' => 'editor.submission.decision.sendExternalReview',
-				'image' => 'advance',
-				'titleIcon' => 'modal_review',
+				'toStage' => 'editor.review',
```
Comment (bozana) [inline classes/workflow/EditorDecisionActionsManager.inc.php]: I believe that this is not needed here, as it is in OMP (when the submission is in the submission stage)? -- It seems to be used only if there are two steps, in promoteForm.tpl... Maybe later this decision action can be implemented in the same way, in two steps (first authors notification and then file selection)?

File: classes/workflow/EditorDecisionActionsManager.inc.php [hunk 2]
```diff
@@ -164,33 +158,17 @@ static function _externalReviewStageDecisions() {
 				'operation' => 'sendReviewsInReview',
 				'name' => 'requestRevisions',
 				'title' => 'editor.submission.decision.requestRevisions',
-				'image' => 'revisions',
-				'help' => 'editor.review.NotifyAuthorRevisions',
-				'titleIcon' => 'revisions_required',
-			),
-			SUBMISSION_EDITOR_DECISION_RESUBMIT => array(
```
Comment (bozana) [inline classes/workflow/EditorDecisionActionsManager.inc.php]: Maybe to leave this -- it is a valid decision -- but to leave out the 'operation' index (or so) and to always check if  that 'operation' index exist i.e. for actions to be displayed, calculated here https://github.com/pkp/pkp-lib/blob/master/pages/workflow/PKPWorkflowHandler.inc.php#L293 (I am not sure if also somewhere else this was used) ?

Comment (bozana) [reply classes/workflow/EditorDecisionActionsManager.inc.php]: Ah now I see how it is used i.e. where everywhere... Sorry!

Comment (NateWr) [reply classes/workflow/EditorDecisionActionsManager.inc.php]: I got an email with another comment saying this removal is ok, but I don't see that here. Did the comment get deleted? Is it still an issue?

File: classes/workflow/EditorDecisionActionsManager.inc.php (hunk 1 above)
Comment (NateWr) [reply classes/workflow/EditorDecisionActionsManager.inc.php]: I use the `toStage` parameter in the name of the second form part in `promoteForm.tpl`:

https://github.com/pkp/pkp-lib/pull/2733/files/0d17226923c0cf8ddc6954ac7deaa7418d53cf48#diff-033e78a65cd87eb957a71403cfa7a279R75

**Comment by bozana** [comment]
've rebased this in order to be able to merge and thus had to make a new PR: https://github.com/pkp/ojs/pull/1537. Closing this one...

**Pull request closed without merging**